changelog/_unreleased/2020-09-17-optimise-requests-in-localetolanguageservice.md

```diff
@@ -0,0 +1,9 @@
+---
+title:              Optimise requests in LocaleToLanguageService
+issue:              NEXT-10928
+author:             Hannes Wernery
+author_email:       hannes.wernery@pickware.de
+author_github:      @hanneswernery
+---
+# Administration
+* Optimise the requests in `LocaleToLanguageService` so send a single request when calling `localeToLanguage()`.
```

Comment:
Please reword with "* Changed ..."

Reply:
I changed the description, but not the title. Is that correct?